### PR TITLE
chore: use `i18next` vue package

### PR DESCRIPTION
**Pull request by chmelevskij**
This PR is to add better localisation compatibility between old codebase and new vue components.

### CODE REVIEW

**Comment by chmelevskij** [comment]
#2193 @McGiverGim will add formatter here for better idea on what I meant. I think we could just use i18next as it is. but the chrome localisation should be replaced asap as well.

File: src/components/vueI18n.js [hunk 1]
```diff
@@ -1,18 +1,10 @@
 import Vue from "vue";
-import VueI18n from "vue-i18n";
+import VueI18Next from "@panter/vue-i18next";
 
-Vue.use(VueI18n);
+Vue.use(VueI18Nex);
```
Comment (McGiverGim) [inline src/components/vueI18n.js:4]: t missing 😁

**Comment by chmelevskij** [comment]
Closing in favour of #2237

**Comment by chmelevskij** [comment]
Closing in favour of #2237

**Pull request closed without merging**